vm-transfer/zvm-transfer.py

```diff
@@ -0,0 +1,455 @@
+    manageable_volumes, existing_volumes = check_volumes_in_dest(vm, dst_symp_client)
+
+    create_new_vm(vm, networks, manageable_volumes, existing_volumes, dst_symp_client)
+
+    logger.info("VM %s cloning is complete", vm.name)
+
+
+def get_vm_by_id(client, vm_id, vm_list=None):
+    if vm_list is None:
+        vm_list = client.vms.list(detailed=True)
+    vm = [vm for vm in vm_list if vm.id == vm_id]
+    if not vm:
+        logger.info("VM with id: %s does not exists", vm_id)
+        return None
+    return vm[0]
+
+
+def get_vm_by_name(client, vm_name, vm_list=None):
+    if vm_list is None:
+        vm_list = client.vms.list(detailed=True)
+    vm = [vm for vm in vm_list if vm.name == vm_name]
+    if not vm:
+        logger.info("VM with name: %s does not exists", vm_name)
+        return None
+    return vm[0]
+
+
+def init_src_symp_client():
+    my_session = requests.Session()
+    my_session.verify = False
+    client = symphony_client.Client(url='https://%s' % config.SRC_CLUSTER_IP, session=my_session)
+    client.login(domain=config.SRC_ACCOUNT,
+                 username=config.SRC_USERNAME,
+                 password=config.SRC_PASSWORD,
+                 project='default',
+                 mfa_secret=config.SRC_MFA_SECRET)
+    return client
+
+
+def init_dst_symp_client():
+    my_session = requests.Session()
+    my_session.verify = False
+    client = symphony_client.Client(url='https://%s' % config.DST_CLUSTER_IP, session=my_session)
+    client.login(domain=config.DST_ACCOUNT,
+                 username=config.DST_USERNAME,
+                 password=config.DST_PASSWORD,
+                 project='default',
+                 mfa_secret=config.DST_MFA_SECRET)
+    return client
+
+
+def extract_vm_info(vm, src_client):
+    logger.info("Extracting info from VM:\n%s", pformat(unmunchify(vm)))
+    vm_info = dict()
+    vm_info['name'] = vm['name']
+    vm_info['disable_delete'] = vm['disable_delete']
+    vm_info['hw_firmware_type'] = vm['hw_firmware_type']
+    vm_info['instanceType'] = vm['instanceType']
+    vm_info['instance_profile'] = vm['instance_profile']
+    vm_info['key_pair'] = vm['key_pair']
+    vm_info['metadata'] = vm['metadata']
+    vm_info['provided_os_type_id'] = vm['provided_os_type_id']
+    vm_info['ramMB'] = vm['ramMB']
+    vm_info['restart_on_failure'] = vm['restart_on_failure']
+    vm_info['vcpus'] = vm['vcpus']
+    vm_info['ports'] = vm['ports']
+    vm_info['tags'] = [tag for tag in vm['tags'] if not tag.startswith('system:')]
+
+
+    #vm_info['project_id'] = get_dst_project_id(vm['project_id'], src_client, dst_client)
+    #vm_info['bootVolume'] = get_dst_boot_volume_id(vm['bootVolume'], src_client, dst_client)
+    #vm_info['volumes'] = get_dst_volumes_ids(vm['volumes'], src_client, dst_client)
+    #vm_info['imageId'] = get_dst_image_id(vm['imageId'], src_client, dst_client)
+    #vm_info['networks'] = get_dst_networks(vm['networks'], src_client, dst_client)
```

Comment:
All of the commented out code in lines 125-129 can be deleted.

Reply:
I'm not sure we need this any more - not sure we actually using vm_info anywhere 
the code usually access the VM directly

Reply:
In that case, you should delete the entire method.